test/java/seedu/addressbook/data/AddressBookTest.java

```diff
@@ -135,4 +133,39 @@ public void getAllPersons() throws Exception {
 
         assertTrue(isIdentical(allPersons, personsToCheck));
     }
+
+    @Test
+    public void getSortedList() throws Exception {
+        UniquePersonList testPersonList  = new UniquePersonList(bobChaplin, aliceBetsy);
+        List<Person> unsortedList = testPersonList.listView();
+        List<Person> sortedList = testPersonList.sortedListView();
+        assertListNotSorted(unsortedList);
+        assertListSorted(sortedList);
+    }
+
+    public void assertListSorted(List<Person> inputList) {
+        List<Person> sortedList = sortList(inputList);
+        assertTrue(isIdentical(inputList, sortedList));
+    }
+
+    public void assertListNotSorted(List<Person> inputList) {
+        List<Person> sortedList = sortList(inputList);
+        assertFalse(isIdentical(inputList, sortedList));
+    }
+
+    public List<Person> sortList(List<Person> inputList) {
+        List<Person> outputList = new ArrayList<>();
+        for (Person p : inputList) {
+            outputList.add(p);
+        }
+        Collections.sort(outputList, new Comparator<Person>() {
+            @Override
+            public int compare(Person person1, Person person2) {
+                String name1 = person1.getName().fullName;
+                String name2 = person2.getName().fullName;
+                return name1.compareToIgnoreCase(name2);
+            }
+        });
+        return outputList;
+    }
```

Comment:
You have already implemented this method somewhere else in the code base... can you reuse that implementation instead of writing a new method here?